database/migrations/Social/2023_06_15_170518_users_lists_messages.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::connection('social')->create('users_lists_messages', function (Blueprint $table) {
+            $table->id();
+            $table->integer('users_lists_id');
+            $table->bigInteger('messages_id');
```

Comment:
@FredPeal add created_At , udpated_At and weight so people can move things

Reply:
$table->timestamps();
add created_at and updated_at